prov/util/src/util_shm.c

```diff
@@ -97,12 +98,18 @@ int smr_create(const struct fi_provider *prov, struct smr_map *map,
 
 	tx_size = roundup_power_of_two(attr->tx_count);
 	rx_size = roundup_power_of_two(attr->rx_count);
+
+	//TODO - find better way to calculate number of SAR buffers
+	//(at least 1 but can't be outrageous)
+	sar_cnt = MAX(1, rx_size / 4);
```

Comment:
Is there any reason we user rx_size/4 here? IIRC, the rx_size is hard-coded to 1024, then why we need to compare with 1?

Reply:
This calculation and the comment above are outdated from earlier changes when peers could have more than 1 pending SAR transfer at a time.
Since the maximum number of peers is hard coded to 256 right now and every peer is only allowed to have one outstanding SAR transfer at a time, we will only ever need SMR_MAX_PEERS number of SAR buffers. We can change it to SMR_MAX_PEERS instead. Does that seem reasonable to you?

Reply:
Yup, sounds reasonable to me. Please add the comment when you change it, so users can get a little context.